### PR TITLE
GDG Lisbon 2016 added

### PR DESCRIPTION
**Pull request by smithereen**
Thank you so much for your work on this template, it really helped the GDGLisbon DevFest.

### CODE REVIEW

**Comment by ozasadnyy** [comment]
Thanks, @smithereen , I wish you will have a great time at DevFest Lisbon!
Please add back spaces at the end of lines, they should be there :)

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ pull request, so we can include you in this list.
 | [Google I/O Extended 2016 Kuala Lumpur](https://ioxkl2016.firebaseapp.com) |
 | [GDG DevFest Hamburg 2016](https://hamburg.devfest.de) |
 | [GDG DevFest Mezam 2016](https://devfest-mezam.gdgbambili.xyz) |
+| [GDG DevFest Lisbon 2016](https://devfestlisbon.firebaseapp.com) |
```
Comment (smithereen) [inline README.md]: Should I had backspaces at the end of these lines?

File: README.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ git commit -m 'Updated to the latest version'
 
 ### Who uses the template?
 
-Going to use the template? Go on! The only thing we ask - let us know at with a 
+Going to use the template? Go on! The only thing we ask - let us know at with a
```
Comment (smithereen) [inline README.md]: this change was certainly by an automatic method, should I revert this and the other cases similar?

Comment (ozasadnyy) [inline README.md]: Yeah, all these spaces on the line ends should be there.

**Comment by smithereen** [comment]
Done

**Comment by ozasadnyy** [comment]
Awesome! Thanks a lot!

**Pull request merged**